__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data")
+
+beforeEach(() => seed(testData));
+
+afterAll(() => {
+    return db.end();
+});
+
+describe ("GET /api/topics", () => {
+    test("status 200, returns an array of topic objects", () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then((response) => {
+        expect(response.body.topics).toBeInstanceOf(Array);
+        expect(response.body.topics).toHaveLength(3);
+        response.body.topics.forEach((topic) => {
+            expect(topic).toHaveProperty('slug', expect.any(String));
+            expect(topic).toHaveProperty('description', expect.any(String));
+        })
```

Comment:
Great to see you are checking the length before your forEach 👍  Using jest matchers such as toMatchObject or objectContaining keep the tests flexible if the test data was to change or extend in properties in the future.